releasenotes/notes/disable-leader-elect.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: release-notes/v2
+kind: bug-fix
```

Comment:
This is not bug-fix

Reply:
Thanks, I wasn't sure what values were allowed here, but I saw that almost all others use `bug-fix` or `feature`, so I switched to `feature`.